cogs5e/dice.py

```diff
@@ -50,81 +52,74 @@ async def rollCmd(self, ctx, *, rollStr: str = '1d20'):
         e (explode dice of value)
         ra (reroll and add)
         __Supported Selectors__
+        X (literal X)
         lX (lowest X)
         hX (highest X)
-        >X/<X (greater than or less than X)"""
+        >X (greater than X)
+        <X (less than X)"""
 
         if rollStr == '0/0':  # easter eggs
             return await ctx.send("What do you expect me to do, destroy the universe?")
 
-        adv = 0
-        if re.search('(^|\s+)(adv|dis)(\s+|$)', rollStr) is not None:
-            adv = 1 if re.search('(^|\s+)adv(\s+|$)', rollStr) is not None else -1
-            rollStr = re.sub('(adv|dis)(\s+|$)', '', rollStr)
-        res = roll(rollStr, adv=adv)
-        out = res.result
+        rollStr, adv = self._string_search_adv(rollStr)
+
+        res = roll(rollStr, advantage=adv, allow_comments=True, stringifier=VerboseMDStringifier())
```

Comment:
This now uses [d20.roll](https://d20.readthedocs.io/en/latest/api.html#d20.Roller.roll) to handle all the rolling and string building logic (see VerboseMDStringifier later if you're curious about the string building + https://d20.readthedocs.io/en/latest/api.html#stringifiers)